client/container_copy.go

```diff
@@ -0,0 +1,110 @@
+package client
+
+import (
+	"context"
+	"net/url"
+	"io"
+
+	"github.com/alibaba/pouch/apis/types"
+	"path/filepath"
+	"fmt"
+	"net/http"
+	"encoding/base64"
+	"strings"
+	"encoding/json"
+	"io/ioutil"
+)
+
+// ContainerStatPath returns Stat information about a path inside the container filesystem.
+func (client *APIClient) ContainerStatPath(ctx context.Context, name string, path string) (types.ContainerPathStat, error) {
+	query := url.Values{}
+	query.Set("path", filepath.ToSlash(path))
+	urlStr := fmt.Sprintf("/containers/%s/archive", name)
+
+	response, err := client.head(ctx, urlStr, query, nil)
+	if err != nil {
+		return types.ContainerPathStat{}, err
+	}
+
+	if err != nil {
+		return types.ContainerPathStat{}, err
+	}
+	defer ensureReaderClosed(response)
+	return getContainerPathStatFromHeader(response.Header)
+}
+
+func ensureReaderClosed(response *Response) {
```

Comment:
This function is redudant with https://github.com/alibaba/pouch/blob/master/client/utils.go#L18. 
Maybe we can re-use that.